code/__DEFINES/materials.dm

```diff
@@ -7,6 +7,22 @@
 ///Use this flag on TRUE if you want the basic recipes
 #define MAT_CATEGORY_BASE_RECIPES "basic recipes"
 
+// Breakdown flags used when decomposing alloys.
+/// Can reduce an alloy into its component materials.
+#define BREAKDOWN_ALLOYS				(1<<0)
```

Comment:
Is the idea of this that right now they're all the same but later on we can add more options? Seeing as they all have the same value

Reply:
Pretty much. At some point there should be different methods for breaking down different alloys but at the moment couldn't be bothered to replace the entire recycling system.

Reply:
Sounds fine to me, maybe add a comment that says this to clear up any confusion?

Reply:
Comment added.